### PR TITLE
New Issue ?body= query

### PR DESCRIPTION
**Pull request by KatieFrogs**
Like with mailto email links, Github allows filling the body of the issue with the "body" query string in the URL, you can see it here:
https://github.com/LoveEevee/gitea/issues/new?body=This+is+an+example
This is particularly useful to make a link in an application and include a bug reporting button, which pre-fills the issue with a specific template, but also with some debug information that the software author wants to be included with the bug.
This PR implements an optional "body" that may be included with the URL when creating a new issue in a repository.

### CODE REVIEW

**Comment by kolaente** [comment]
Can we put this in the docs?
I'd propose to put a sentence about this in https://docs.gitea.io/en-us/issue-pull-request-templates/

**Comment by KatieFrogs** [comment]
@kolaente I have updated the documentation, thanks for the suggestion.

File: docs/content/doc/usage/issue-pull-request-templates.en-us.md [hunk 1]
```diff
@@ -39,3 +39,6 @@ Possible file names for PR templates:
 * .gitea/pull_request_template.md
 * .github/PULL_REQUEST_TEMPLATE.md
 * .github/pull_request_template.md
+
+
+Additionally, the New Issue page URL can be suffixed with `?body=Issue+Text` and the form will be populated with that string. This string will be used instead of the template if there is one.
```
Comment (kolaente) [inline docs/content/doc/usage/issue-pull-request-templates.en-us.md:44]: Doesn't this also work for pull requests? If so, you should mention that :slightly_smiling_face:

Comment (KatieFrogs) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md:44]: It does not, this PR adds supports only for new issues. Implementing this to new pull requests makes little sense to me, as the developer also has to know which branches the user needs to compare in advance.

Comment (kolaente) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md:44]: Makes sense, thanks for clarifying!

**Review by kolaente — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Pull request merged**